### PR TITLE
community calls: add agenda for 2021-08-10 call

### PR DESCRIPTION
**Pull request by t-lo**
This PR adds the agenda for the upcoming August community call. The PR is currently in Draft mode, the following TODOs are open:
- [x] @t-lo to prepare "News" section
- [x] @join2ashish to add Zoom link (feel free to add the link in a comment to this PR)
- [ ] @jepio to prepare presentation on cgroups v2 / docker 20.10 Spotlight
- [ ] @aniruddha2000 to prepare presentation on Locksmith work (please let us know in the PR comments!)
- [ ] @miao0miao and @wrl to prepare ARM64 status update
- [x] @sayanchowdhury to prepare for release planning section

### CODE REVIEW

**Comment by join2ashish** [comment]
Kinvolk Events is inviting you to a scheduled Zoom meeting.

Topic:  Flatcar Community Call | 2021-08-10
Time: Aug 10, 2021, 05:30 PM Amsterdam, Berlin, Rome, Stockholm, Vienna

Join Zoom Meeting
https://us06web.zoom.us/j/83689202160

Meeting ID: 836 8920 2160
One tap mobile
+12532158782,,83689202160# US (Tacoma)
+13017158592,,83689202160# US (Washington DC)

Dial by your location
        +1 253 215 8782 US (Tacoma)
        +1 301 715 8592 US (Washington DC)
        +1 312 626 6799 US (Chicago)
        +1 346 248 7799 US (Houston)
        +1 669 900 6833 US (San Jose)
        +1 929 205 6099 US (New York)
Meeting ID: 836 8920 2160
Find your local number: https://us06web.zoom.us/u/kxaFY3Tx4

**Review by jepio — COMMENTED** [review]
Looks good.

File: community-meetings/2021-08-10.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- Brief intro / check-in of all participants in the Zoom call. Please introduce yourself and share what brings you here today.
+- Review the meeting agenda.
+
+## News
+- New section! We briefly discuss news and happenings in the Flatcar world.
+
+## Spotlight: Docker 20.10 / cgroups v2 update
+- Docker 20 and CGroups v2 (unified mode) are coming to Flatcar Linux! We'll give an overview of the challenges and the implications, as well as discuss the timeline.
+
+## Spotlight community committer: fixing the Locksmith reboot issue
```
Comment (tormath1) [inline community-meetings/2021-08-10.md]: As mentioned last week, I think the spotlight will be more around :arrow_down: 
```suggestion
## Spotlight community committer: contributing to Flatcar
```

and not dedicated to the locksmith issue. :)

**Review by tormath1 — APPROVED** [review]
LGTM :)

**Pull request merged**